README.md

```diff
@@ -13,11 +13,11 @@ Most of the code is in a similar format to the [plutus-pioneer-program](https://
 
 ## Installation
 
-1. Install Python 3.8.
-Installer [download](https://www.python.org/downloads/release/python-3810/)
+1. Install Python 3.8, 3.9 or 3.10 (if it not already installed on your operating system).
+Python3.8 Installer [download](https://www.python.org/downloads/release/python-3810/)
 
-2. Ensure `python3.8 --version` works in your command line.
-In Windows, you can do this by copying the `python.exe` file to `python3.8.exe` in your `PATH` environment variable.
+2. Ensure `python3 --version` works in your command line and outputs one of the above versions.
```

Comment:
Can you change this to `python3.x` where `x` is the minor version? The reason for this is because poetry needs this in PATH in order to use the `poetry env use 3.x`, which sets the python version for the installed virtual environment.

Reply:
Is this a windows thing? I ran the steps locally and it worked fine, but I did actually not try to run `poetry env use`.
`poetry env use` is also not mentioned anywhere in the intro, should that be documented?

The overall goal is anyways that the user should not really have to worry about the version of python they are running anymore (as long as it is  > 3.7)

Reply:
It is just Windows as poetry looks for python versions in path but windows doesn't do this like Linux package managers

Reply:
`poetry env use` is completely optional, just if you want to specify which version of python to use. Otherwise poetry uses the latest python version.